lib/active_storage_support/support_for_base64.rb

```diff
@@ -0,0 +1,24 @@
+require "active_support/concern"
+require "active_storage/attached"
+
+module ActiveStorageSupport
+  module SupportForBase64
+    extend ActiveSupport::Concern
+    class_methods do
+      def has_one_base64_attached(name, dependent: :purge_later)
```

Comment:
I think that with `lib/active_storage_support/base64_one.rb` you are already overriding ActiveStorage solution. Why do you need this file at all? I feel like you have implemented 2 solutions for the same thing

Reply:
How so? Don't think I understood what you meant here.

Reply:
Nevermind. I thought that you were overriding the `attach` method of `ActiveStoage::Attached::One`. Now I see you created another class `Base64One`.